### PR TITLE
Localization search tags

### PR DESCRIPTION
**Pull request by holyjewsus**
This PR adds:
* additional search tags for zero touch Core.List nodes
* search tags for higher order and lacing nodes (added to resource files)
* search tags for watch3d, watch, and watch image (added to resource files)
* search tags for Core.String zero touch nodes

* also made some changes to BaseType.CS, Watch.Cs and Watch3d where the resource type was pointing to Dynamo.properties instead of the correct Namespace.properties, the symptom was the node description,and tooltips would show as the ID instead of the value in the resource dict.

@kronz @lillismith @lukechurch 
Since this PR changes behavior of search it might be good to pulldown and play with to see if searches produce unexpected results.

screen shot 2015-01-26 at 7 50 04 pm
screen shot 2015-01-26 at 7 50 20 pm
screen shot 2015-01-26 at 7 52 58 pm

### CODE REVIEW

**Comment by holyjewsus** [comment]
@lukechurch  PTAL

**Comment by holyjewsus** [comment]
Ping, @lukechurch  @kronz 

I've also dropped a build here https://www.dropbox.com/sh/65b37oiuurbshn4/AADhQs7NggiombKfpCwRS8pja?dl=0

**Comment by kronz** [comment]
@holyjewsus LGTM after a quick scan through the functionality.   Definitely an improvement.  Thanks

File: src/Libraries/CoreNodeModels/HigherOrder/MapCombineLacing.cs [hunk 1]
```diff
@@ -264,6 +270,7 @@ public TrueForAll()
 
     [NodeName("List.Reduce")]
     [NodeCategory(BuiltinNodeCategories.CORE_LISTS_ACTION)]
+	[NodeSearchTags("ListReduceSearchTags", typeof(DSCoreNodesUI.Properties.Resources))]
```
Comment (lukechurch) [inline src/Libraries/CoreNodeModels/HigherOrder/MapCombineLacing.cs]: Please be consistent with putting the search tags after the description

File: src/Libraries/CoreNodeModels/Input/BaseTypes.cs [hunk 2]
```diff
@@ -15,6 +15,7 @@
 using ProtoCore.Namespace;
 using String = DSCoreNodesUI.String;
 
+
```
Comment (lukechurch) [inline src/Libraries/CoreNodeModels/Input/BaseTypes.cs]: Please revert local changes to this

**Comment by lukechurch** [comment]
Clearly an improvement. LGTM

**Comment by holyjewsus** [comment]
Thanks @lukechurch  and @kronz , will make the changes and merge asap.

**Pull request merged**